### PR TITLE
Docs: Fix Sphinx annotations in Doc/library/ctypes.rst

### PR DESCRIPTION
**Pull request by erlend-aasland**
----
:books: Documentation preview :books:: https://cpython-previews--107672.org.readthedocs.build/

### CODE REVIEW

File: Doc/conf.py [hunk 1]
```diff
@@ -97,6 +97,7 @@
     ('c:func', 'vsnprintf'),
     # Standard C types
     ('c:type', 'FILE'),
+    ('c:type', 'int'),
```
Comment (AA-Turner) [inline Doc/conf.py]: This shouldn't be needed (see other comments)
```suggestion
```

File: Doc/library/ctypes.rst [hunk 2]
```diff
@@ -474,17 +475,17 @@ Return types
    strchr = libc.strchr
 
 
-By default functions are assumed to return the C :c:expr:`int` type.  Other
+By default functions are assumed to return the C :c:type:`int` type.  Other
```
Comment (AA-Turner) [inline Doc/library/ctypes.rst]:
```suggestion
By default functions are assumed to return the C :c:expr:`int` type.  Other
```

File: Doc/library/ctypes.rst [hunk 3]
```diff
@@ -474,17 +475,17 @@ Return types
...
 return types can be specified by setting the :attr:`restype` attribute of the
 function object.
 
-The C prototype of ``time()`` is ``time_t time(time_t *)``. Because :c:type:`time_t`
-might be of a different type than the default return type ``int``, you should
-specify the ``restype``::
+The C prototype of :c:func:`time` is ``time_t time(time_t *)``. Because :c:type:`time_t`
+might be of a different type than the default return type :c:type:`int`, you should
```
Comment (AA-Turner) [inline Doc/library/ctypes.rst]:
```suggestion
might be of a different type than the default return type :c:expr:`int`, you should
```

**Review by AA-Turner — COMMENTED** [review]
It seems `` :c:type:`spam` `` is a cross-reference role, and expects to be able to resolve a name -- it explicitly doesn't let you cross reference a keyword (it seems from the [table here](https://en.cppreference.com/w/c/keyword)). `` :c:expr:`spam` `` is the only role that doesn't fail here.

A

File: Doc/conf.py (hunk 1 above)
Comment (erlend-aasland) [reply Doc/conf.py]: Yeah, that's what I thought. It was a last resort to try and fix the CI. Thanks.

**Comment by erlend-aasland** [comment]
Thanks, Adam!

**Pull request merged**